draft-ietf-taps-impl.md

```diff
@@ -289,31 +289,31 @@ Differences in possible protocol compositions and options can also provide a bra
 This approach is commonly used for connections with optional proxy server configurations. A single connection might have several options available: an HTTP-based proxy, a SOCKS-based proxy, or no proxy. As above, these options should be ranked based on preference, system policy, and performance and attempted in succession.
 
 ~~~~~~~~~~
-1 [www.example.com:80, Any, HTTP/TCP]
-  1.1 [192.0.2.8:80, Any, HTTP/HTTP Proxy/TCP]
-  1.2 [192.0.2.7:10234, Any, HTTP/SOCKS/TCP]
-  1.3 [www.example.com:80, Any, HTTP/TCP]
-    1.3.1 [192.0.2.1:80, Any, HTTP/TCP]
+1 [www.example.com:80, Wi-Fi, HTTP/TCP]
```

Comment:
These examples of "Any" had meant "no interface preference". Changing them all to require Wi-Fi seems wrong?

Reply:
Before reading #1107 I had argued the same, but I guess the "Any" easily distracts from the protocol as its meaning is not obvious on first sight. Having anything non-changing in the examples solves that issue.
As an Alternative, I would suggest replacing all occurrences of `Any` in the path column with `AnyPath` 
Does this work four you @tfpauly?

Reply:
Before reading #1107 I had argued the same, but I guess the "Any" easily distracts from the protocol as its meaning is not obvious on first sight. Having anything non-changing in the examples solves that issue.
As an Alternative, I would suggest replacing all occurrences of `Any` in the path column with `AnyPath` 
Does this work four you @tfpauly?

Reply:
An alternative would be to explain the complete example, mentioning also the endpoint and the meaning of Any for the path. But I am fine with AnyPath.

Reply:
I think I'd prefer using "Any Path" (written out with a space). But that looks good.

Reply:
looks good. After editing, I needed to adjust to lower case for readability and change at a few additional places for consistency.